vowpalwabbit/cb_algs.h

```diff
@@ -69,6 +69,13 @@ inline float get_unbiased_cost(CB::cb_class* observation, COST_SENSITIVE::label&
   return get_unbiased_cost(observation, action);
 }
 
+inline float get_unbiased_cost(ACTION_SCORE::action_score& a_s, float cost, uint32_t action, float offset = 0.)
+{
```

Comment:
The name of this function doesn't make sense unless offset = 0.

Reply:
I simply copied from similar implementations above, what would you suggest?

Reply:
Maybe "cost_estimate"?

Reply:
For the new function? Or to rename existing usage too?

Reply:
It seems more consistent to do both.

Reply:
Sounds good